truffle.js

```diff
@@ -5,14 +5,14 @@ module.exports = {
     development: {
       host: "localhost",
       port: 8545,
-      gas: 4700000,
+      gas: 6000000,
```

Comment:
This is `7000000` in Parity, shall we keep them the same?

Reply:
This change is causing the token test failures as the consumed gas by a failed `assert` is now 93,253 rather than 73,000. This is not the case with `parity`, just with `ganache-cli`. With parity the gas used is always `4627038`